plugin/qrfilter.cpp

```diff
@@ -10,12 +10,6 @@
 QrFilter::QrFilter(QObject *parent)
     : QAbstractVideoFilter(parent)
 {
-    connect(this, &QrFilter::activeChanged, this, [=]{
-        if (!isActive()) {
-            m_result.clear();
-            emit resultChanged(m_result);
-        }
-    });
```

Comment:
It is necessary to remove auto clearing on changing active state?

Reply:
Not necessary but started thinking it might be a good idea to give app the control, and the same behavior is easy to do.

Reply:
One line on the camera.

Actually, ponder if the app side should deactivate the detection when it gets a result. If the filter cleared the result automatically that could get more difficult.